README.th.md

```
@@ -271,86 +262,89 @@ module.CanPierce = function(cast : TypeDef.ActiveCastData, resultOfCast : Raycas
-- [EtiTheSpirit](https://github.com/EtiTheSpirit): นักพัฒนาต้นฉบับ
-- [Per2iako](https://github.com/Per2iako): แก้ ActivesRef ถูกเขียนทับ (BaseParallel, ParallelSimulation)
+# ผู้อยู่เบื้องหลัง FastCast2 (Contributors)
+- [CK06](https://github.com/weenachuangkud): นักพัฒนาหลัก, ผู้ดูแลโปรเจกต์, นักออกแบบกราฟิก
+- [Naymmmm](https://github.com/Naymmmm): ผู้ดูแลโปรเจกต์ (ไม่ได้ใช้งานแล้ว)
+- [EtiTheSpirit](https://github.com/EtiTheSpirit): ผู้พัฒนาต้นฉบับ
+- [Per2iako](https://github.com/Per2iako): ผู้ดูแลโปรเจกต์
+
+# ขอบคุณเป็นพิเศษ
 
-# ขอบคุณพิเศษ
+### ขอบคุณเป็นพิเศษแก่ผู้คนต่อไปนี้จาก [เซิร์ฟเวอร์ Discord ของ Suphi Kaner](https://youtube.com/@5uphi?si=gWWZ6RNqaEYx26az):
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)